lib/network.go

```diff
@@ -174,9 +174,9 @@ func (msgType MsgType) String() string {
 		return "GET_ADDR"
 	case MsgTypeQuit:
 		return "QUIT"
-	case MsgTypeNewPeer:
-		return "NEW_PEER"
-	case MsgTypeDonePeer:
+	case MsgTypeAcceptedPeer:
+		return "ACCEPTED_PEER"
```

Comment:
Are there any side effects to changing the msg type's String func behavior? Should we also update the case below for `MsgTypeDisconnectedPeer`?